src/webview/memory-webview-view.tsx

```diff
@@ -82,7 +82,7 @@ const MEMORY_DISPLAY_CONFIGURATION_DEFAULTS: MemoryDisplayConfiguration = {
     showRadixPrefix: true,
 };
 
-const DEFAULT_READ_ARGUMENTS: Required<ReadMemoryArguments> = {
+export const DEFAULT_READ_ARGUMENTS: Required<ReadMemoryArguments> = {
```

Comment:
Would it make sense to move this into a more common place? Like `../common/messaging` or `../common/memory`? Looks like `ReadMemoryArguments` is defined in `../common/messaging`.

Reply:
IMO I don't see any benefit in doing that.
The default constants defined in the `memory-webview-view` (MEMORY_DISPLAY_CONFIGURATION_DEFAULTS,DEFAULT_SESSION_CONTEXT etc.) only serve the purpose of constructing the initial state for the `App` component. They are an implementation detail of ( and scoped to) the webview/react app and should not leak into other components.

Reply:
I agree with @jreineckearm that it would be better to move this to a constants file (e.g. `view-types.ts`) to avoid circular dependencies.